index.html

```diff
@@ -3,19 +3,87 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
+    <title>Scoala de Snowboarding</title>
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
-    <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
 
+
+<!--HEADER-->
+
+<nav id="page-nav">
+    <div class="nav-bar">
+    <a href="index.html"><img src="/img/karma-logo.svg" alt=""></a>
+    
+    
+    <!-- [THE HAMBURGER] -->
+    <label for="hamburger">&#9776;</label>
+    <input type="checkbox" id="hamburger"/>
+
+    <!-- [MENU ITEMS] -->
+    <ul>
+      <li><a class="active" href="#" >Home</a></li>
+      <li><a href="#">About</a></li>
+      <li><a href="#">Contact</a></li>
+    </ul>
+    </div>
+  </nav>
+<!--MAIN-->
+
+    <main id="back">
+        <div id="title">
+            <h1>Snow School</h1>
+            <p>Snowboard, Ski, Fun</p>
+            <button id="fbtn">APPLY HERE!</button>
+        </div>
+    </main>
+<!--FEATURES-->
+    <section>
+        <h2>Everyone needs a little Karma.</h2>
+
+    <div id="features">
+        <div class="img-features">
+            <img src="img/icon-coffee.svg" alt="">
+            <p>Internet for all devices</p>
+        </div>
+        <div class="img-features">
+            <img src="img/icon-devices.svg" alt="">
+            <p>Boost your productivity</p>
+        </div>
+        <div class="img-features">
+            <img src="img/icon-refill.svg" alt="">
+            <p>Pay as You Go</p>
+        </div>
+       
+    </div>
+    <hr>
+
+    <footer>
+        <p>Join us on </p>
+        <div id="social">
+            <a href="#" class="fa fa-facebook"></a>
+            <a href="#" class="fa fa-twitter"></a>
+            <a href="#" class="fa fa-instagram"></a>
+            
+        </div>
+       <p>&copy; Alexandru Budaca</p> 
+    </footer>
+   
+   
+
+
+    </section>
```

Comment:
This `<section>` wraps part of the main body of the page, and also the footer. So your whole 'features' section is neither in `main` nor in `footer`.
Try moving the section to be inside the `<main>` and keep the footer separate?